src/templates/Studio/Wonder/Sun/Sun.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import {
+  Animation,
+  DirectionalLight,
+  Color3,
+} from 'babylonjs';
+
+const Sun = ({ layout, onAddSun, scene, world }) => {
+  useEffect(() => {
+    let interval;
+    let light;
+    let prevProgress = 0;
+
+    const getMinutesSinceMidnight = (timestamp = Date.now()) => {
+      const date = new Date(timestamp);
+      const utcDate = date.getTime() + (date.getTimezoneOffset() * 60000);
+      const amsterdamDate = new Date(utcDate + (3600000 * 1));
```

Comment:
` * 1` could go. Also there might be cleaner ways to get the local time, but this looks simple enough.